_posts/2016-12-02-november.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: "Dev Diary (November 2016)"
+author: "Corey Montella"
+tags: []
+---
+
+_(Eve is a new programming language, and this is our development blog. If you’re new to Eve, [start here](http://play.witheve.com))_
+
+With the launch of Eve v0.2 at the end of October, the early weeks of November were spent reacting and responding to feedback from new users to the ever-growing Eve community.
+
+### What's new in Eve Alpha v0.2.2?
+
+For a full list of the changes, check out the [release notes][1.1]. We've got several much-requested features in there for you. Let's take a look at those in detail.
+
+#### Eve npm package
+
+We published a package for Eve on npm, so you can now install Eve like so:
+
+```
+npm install -g witheve
+``` 
+
+This will give you a global Eve installation that you can invoke with the command `eve` from anywhere. Doing so will start an Eve server, in the same way `npm start` currently does.
+
+#### Workspaces
+
+We're introducing the ability to run Eve in a custom project workspace. To create a new Eve project workspace, just create a folder with an empty file named `package.json`. Now, start Eve from within this folder. Eve recognizes that it's starting in an Eve project, and will serve *.eve files from within this directory instead of the Eve examples folder. 
+
+Furthermore, you can serve various assets for your Eve programs. Eve currently looks for the following folders in your project, and will serve files from them:
+
+- `/css` - for serving stylesheets
+- `/assets` - for serving images and other binary assets
+- `/fonts` - for serving custom fonts 
+
+#### Server Mode
+
+Now, Eve can be started in server mode using the `--server` flag:
+
+```
+eve --server
+```
+
+Without this flag, execution of Eve programs happens within the browser, with the Eve server acting as an intermediary between the browser and your local system. In server mode, Eve will instead execute your program on the server. Currently written programs will operate exactly as before, but this is a preliminary step in order to get networked Eve applications going (like a chat server, or a multiplayer game). There is still work needed to be done there (currently [in review](https://github.com/witheve/Eve/pull/620)), so stay tuned for more!
+
+#### Deploying Eve Apps
+
+This is one of the most requested features to date, so we're happy to bring it to you. You can now run Eve applications without the editor, which should pave the way to deploying them on your own server. You can specify which Eve program you want to run after the Eve command:
+
+```
+eve ~/myEveDir/clock.eve
+```
+
+Now you can navigate to Eve in your browser to access the specified program. If you like, you can also recover the editor with a flag:
+
+```
+eve ~/myEveDir/clock.eve --editor
+```
+
+This will run the clock program with the editor visible.
```

Comment:
"A program named clock.eve" would be better, so people don't wonder if this somehow magically aliases examples.